src/notebooklm/cli/notebook.py

```
@@ -94,6 +99,7 @@ async def _run():
                     return
 
                 console.print(f"[green]Created notebook:[/green] {nb.id} - {nb.title}")
+                console.print(f"[dim]Context set to new notebook[/dim]")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)